fn_falcon_sandbox/fn_falcon_sandbox/components/falcon_sandbox_submit_file.py

```diff
@@ -0,0 +1,229 @@
+            falcon_sandbox_environment_variable = kwargs.get("falcon_sandbox_environment_variable")  # text
+            falcon_sandbox_custom_run_time = kwargs.get("falcon_sandbox_custom_run_time")  # text
+            falcon_sandbox_submit_name = kwargs.get("falcon_sandbox_submit_name")  # text
+            falcon_sandbox_custom_date_time = kwargs.get("falcon_sandbox_custom_date_time")  # text
+            falcon_sandbox_document_password = kwargs.get("falcon_sandbox_document_password")  # text
+            falcon_sandbox_tor_enabled_analysis = kwargs.get("falcon_sandbox_tor_enabled_analysis")  # boolean
+            falcon_sandbox_incident_id = kwargs.get("falcon_sandbox_incident_id")  # number
+            falcon_sandbox_task_id = kwargs.get("falcon_sandbox_task_id")  # number
+            falcon_sandbox_attachment_id = kwargs.get("falcon_sandbox_attachment_id")  # number
+            falcon_sandbox_artifact_id = kwargs.get("falcon_sandbox_artifact_id")  # number
+
+            log = logging.getLogger(__name__)
+            log.info("falcon_sandbox_environment: %s", falcon_sandbox_environment)
+            log.info("falcon_sandbox_action_script: %s", falcon_sandbox_action_script)
+            log.info("falcon_sandbox_no_share_third_party: %s", falcon_sandbox_no_share_third_party)
+            log.info("falcon_sandbox_allow_community_access: %s", falcon_sandbox_allow_community_access)
+            log.info("falcon_sandbox_comment: %s", falcon_sandbox_comment)
+            log.info("falcon_sandbox_priority: %s", falcon_sandbox_priority)
+            log.info("falcon_sandbox_environment_variable: %s", falcon_sandbox_environment_variable)
+            log.info("falcon_sandbox_custom_run_time: %s", falcon_sandbox_custom_run_time)
+            log.info("falcon_sandbox_submit_name: %s", falcon_sandbox_submit_name)
+            log.info("falcon_sandbox_custom_date_time: %s", falcon_sandbox_custom_date_time)
+            log.info("falcon_sandbox_document_password: %s", falcon_sandbox_document_password)
+            log.info("falcon_sandbox_tor_enabled_analysis: %s", falcon_sandbox_tor_enabled_analysis)
+            log.info("falcon_sandbox_incident_id: %s", falcon_sandbox_incident_id)
+            log.info("falcon_sandbox_task_id: %s", falcon_sandbox_task_id)
+            log.info("falcon_sandbox_attachment_id: %s", falcon_sandbox_attachment_id)
+            log.info("falcon_sandbox_artifact_id: %s", falcon_sandbox_artifact_id)
+
+            API_KEY = self.options.get("falcon_sandbox_api_key")
+            API_HOST = self.options.get("falcon_sandbox_api_host")
+            FETCH_REP_TIMEOUT = int(self.options.get("fetch_report_timeout"))
+            FETCH_REP_STAT_INTRVL = int(self.options.get("fetch_report_status_interval"))
+            
+            ## This condition will prevent quering Falcon Sandbox for Status in small intervals
+            if FETCH_REP_STAT_INTRVL and FETCH_REP_STAT_INTRVL:
```

Comment:
Could you explain this if 
It looks like you are testing the truth value of the same thing twice 
Is this the same thing 
```suggestion
            if FETCH_REP_STAT_INTRVL:
```

Reply:
Or maybe you meant something like 

```suggestion
            if FETCH_REP_STAT_INTRVL and FETCH_REP_TIMEOUT:
```

?